extensions/wikia/FounderEmails/FounderEmailsEvent.class.php

```diff
@@ -100,14 +109,6 @@ public static function register() {
 		return true;
 	}
 
-	protected function isThresholdMet( $testValue ) {
```

Comment:
Useless method obviated by removing the 'threshold' parameter from the ```wg``` variable for this extension.